packages/vde2/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/virtualsquare/vde-2
+TERMUX_PKG_DESCRIPTION="Virtual Distributed Ethernet for emulators like qemu"
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_LICENSE_FILE="COPYING"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.3.2
+TERMUX_PKG_GIT_BRANCH=master
```

Comment:
Never specify `master` or any other branch which does not describe package version.

Specify git tag there.